pymc3/tests/test_advi.py

```diff
@@ -83,13 +83,13 @@ def test_check_discrete_minibatch():
 
     def create_minibatch():
         while True:
-            return disaster_data
+            return disaster_data,
```

Comment:
Can you change this to (data,) then? For reasons of style. That way it's obvious to everyone that it's a tuple of length one.